tools/appimage/overlay/usr/share/metainfo/org.randovania.randovania.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd prefer if this file would be generated based on the rest. Particularly since it duplicates part of the README and is in a path that will be absolutely hidden.

Reply:
I'm onboard. Should I parse out a section of the README or include a chunk of data in the build script?

There's quite a bit of boilerplate here; realistically we only need to worry about generating the summary and description.

Reply:
conveniently, the README is already formatted in such a way to facilitate pulling individual sections out of it. there are comments marking various sections: the description tag in this file is in the WELCOME section. worth noting that you'd need to convert the markdown to XML, ofc

Reply:
Added in 0d4af72a